openrpc.json

```diff
@@ -283,6 +283,73 @@
         }
       ]
     },
+    {
+      "name": "wallet_revokePermissions",
+      "tags": [
+        {
+          "$ref": "#/components/tags/MetaMask"
+        }
+      ],
+      "summary": "Revoke existing permissions.",
+      "description": "Revoke permissions from the user",
+      "paramStructure": "by-name",
+      "params": [
+        {
+          "name": "accounts",
```

Comment:
permissions are generic so this cant be called `accounts`. 

it should probably match the requestPermissions where its:

```
interface PermissionRequest {
  [methodName: string]: {
    [caveatName: string]: any;
  };
}
```

Reply:
@shanejonas @danfinlay good catch! I've just implemented the latest changes to the method interface, aligning it perfectly with the specifications outlined in[ EIP-2255](https://eips.ethereum.org/EIPS/eip-2255). It's worth mentioning that MetaMask controller offer support for a single parentCapability, namely "eth_accounts," and a single caveatType, namely "restrictReturnedAccounts."  I have updated the JSON RPC specs to enforce the use of an enum, thereby avoiding users from providing any sort of arbitrary unfiltered input. This update will help make the method interface more compliant with our general-purpose wallet permissioning system.

Reply:
I would also like to share a reference to the wallet revoke permission handler in the MetaMask extension, which can be found at https://github.com/MetaMask/metamask-extension/pull/19444. Your feedback and insights are greatly appreciated.

Reply:
this should look more like this:

https://github.com/MetaMask/metamask-improvement-proposals/pull/24#discussion_r1394485055